File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex5.java
//Importando Scanner
import java.util.Scanner;

public class Ex5{
	
	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Produto dos números digitados\n\n");
		
		//vars
		double num, produto=1;
		
		//Entrada
		System.out.print("Insira um número positivo, ou zero para terminar: ");
		num = ent.nextDouble();
		
		while( num != 0 )
		{
			//Processamento
			produto *= num;
			
			//Entrada
			System.out.print("Insira um número positivo, ou zero para terminar: ");
			num = ent.nextDouble();
			
		}
		
		System.out.printf("Produto = %.2f", produto);
		
		
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex5.c
#include <stdio.h>

main()
{
    printf("Produto dos numeros digitados\n\n");

    //vars
    float num, produto=1;

    //Entrada
    printf("Insira um numero: ");
    scanf("%f", &num);

    //Processamento
    while (num != 0)
    {
        produto *= num;

        printf("Insira um numero: ");
        scanf("%f", &num);
    }

    //Saída
    printf("\nProduto = %.2f", produto);

}
